frontend/common/crud/useGet.tsx

```diff
@@ -28,7 +28,9 @@ export function useGet<T>(url: string, query?: Record<string, string | number |
     url += '?' + new URLSearchParams(normalizedQuery).toString();
   }
 
-  const response = useSWR<T>(url, getRequest);
+  const response = useSWR<T>(url, getRequest, {
+    dedupingInterval: 0,
```

Comment:
Based on SWR docs it looks like this value is used to prevent duplicate requests within a certain time interval with a default interval being 2000 ms. It seems like this code will allow an immediate duplicate request to occur.  Can you expand on this line, and what we are trying to achieve with it?

Reply:
Our code is clean enough that we do not run into an issue when we need dedep of network calls.
In the case of tests, where a test gets an item, edits an item, goes to details, which get the item,
All in under 2000ms, the edited item is not gotten by SWR because of the dedup, so we turn it off.